README.md

```diff
@@ -1,2 +1,23 @@
-# hex2nix
-A tool to automatically convert Hex packages to the Nix Expression format
+
+hex2nix
+=======
+
+hex2nix is a command that generates correct Nix Expressions from the
+Hex.pm repository. It only generates those expressions for packages
+that have a buildable Build System. Right now that is `rebar`,
+`rebar3`, or `make`. It builds all of these with `rebar3`. That works
+for the vast majority of packages but not all.
+
+## Running `hex2nix`
+
+The `hex2nix` command takes one argument. The directory where it will
```

Comment:
It takes one or two args, it's worth keeping it in sync with "usage" message of the script IMO